File: Foundation/Scratch/Getting Started/c++/l001gradingSystem.cpp
#include<iostream>
using namespace std;
void gradingSystem(int m)
{
    if(m > 90)
        cout<<"excellent";
    else if(m > 80)
        cout<<"good";
    else if(m > 70)
        cout<<"fair";
    else if(m > 60)
        cout<<"meets expectations";
    else    
        cout<<"below par";
}
int main()
{
    int marks;
    cin>>marks;
    gradingSystem(marks);
    return 0;
}

File: Foundation/Scratch/Getting Started/java/l001gradingSystem.java
import java.util.Scanner;
public class l001gradingSystem{
    public static Scanner scn = new Scanner(System.in);
    public static void gradingSystem(int m)
    {
        if(m > 90)
            System.out.println("excellent");
        else if(m > 80)
            System.out.println("good");
        else if(m > 70)
            System.out.println("fair");
        else if(m > 60)
            System.out.println("meets expectations");
        else
            System.out.println("below par");
    }
    public static void main(String[] args)
    {
        int marks = scn.nextInt();
        gradingSystem(marks);
    }
}
